generators/controller/templates/app/src/main/java/model/response/Response.java

```diff
@@ -0,0 +1,21 @@
+package <%= packageName %>.model.response;
+
+import <%= packageName %>.entities.<%= entityName %>;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class <%= entityName %>Response {
```

Comment:
Instead of Entity specific class, can we have a generic response like https://github.com/sivaprasadreddy/devzone/blob/main/application/src/main/java/com/sivalabs/devzone/common/PagedResult.java ?

Reply:
If there are many entities then how to handle it? Example customer, order, etc.. that is the reason I have chosen entity name

Reply:
It makes sense, I will make necessary changes

Reply:
> Instead of Entity specific class, can we have a generic response like https://github.com/sivaprasadreddy/devzone/blob/main/application/src/main/java/com/sivalabs/devzone/common/PagedResult.java ?

Implemented using records , to get flavor of java 17. Please review